MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
Wait, why are gamedev commits from 10 days ago in this PR? These commits are already in master